Dockerfile

```diff
@@ -20,10 +20,20 @@ COPY . .
 
 ENV MIX_ENV=prod
 
-RUN mix do deps.get, deps.compile, compile
+# Install and compile dependencies
+RUN mix do deps.get, deps.compile
 
+# Build assets
+COPY assets/package.json assets/package-lock.json ./assets/
+RUN npm --prefix ./assets ci --progress=false --no-audit --loglevel=error
 
-RUN mix release
+COPY priv priv
+COPY assets assets
```

Comment:
I don't think these `COPY` commands are needed since all files should have been copied already with `COPY . .` on line 19.